src/linker/Linker/DirectoryAssemblyResolver.cs

```diff
@@ -89,6 +121,13 @@ public void Dispose ()
 
 		protected virtual void Dispose (bool disposing)
 		{
+			if (disposing) {
```

Comment:
Should there be also finalizer?

Reply:
The class is not holding unmanaged resources so I don't think we need one.